<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgFilme</class>
 <widget class="QDialog" name="dlgFilme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Código:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="tbxCodigo"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Título:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="5">
      <widget class="QLineEdit" name="tbxTitulo"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ano:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="5">
      <widget class="QLineEdit" name="tbxAno"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Prod.:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="5">
      <widget class="QLineEdit" name="tbxProdutora"/>
     </item>
     <item row="5" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mídia:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="8">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Diretor:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="5">
      <widget class="QLineEdit" name="tbxDiretor"/>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ator P.:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3" colspan="5">
      <widget class="QLineEdit" name="tbxAtorPrincipal"/>
     </item>
     <item row="8" column="6">
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Estilo:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <widget class="QComboBox" name="cbMidia">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Selecione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DVD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blu ray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nuvem</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fita</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="6" colspan="2">
      <widget class="QComboBox" name="cbEstilo">
       <property name="font">
        <font>
         <family>Yu Gothic Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Selecione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Terror</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Drama</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Romance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Suspense</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aventura</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ação</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Animação</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Documentário</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
